### PR TITLE
Migrate iOS test app to react-native-test-app

### PR DESCRIPTION
**Pull request by tido64**
### Platforms Impacted
- [x] iOS
- [ ] macOS
- [ ] win32
- [ ] windows
- [ ] android

### Description of changes

Migrates iOS test app to `react-native-test-app`.

### Verification

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Simulator Screen Shot - iPhone 11 Pro Max - 2020-06-23 at 19 52 18 | Simulator Screen Shot - iPhone 11 Pro Max - 2020-06-23 at 19 52 46 |

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

cc @FalseLobster, @HeyImChris

### CODE REVIEW

File: apps/ios/react-native.config.js [hunk 1]
```diff
@@ -18,7 +18,7 @@
 module.exports = {
   project: {
     ios: {
-      project: "apps/ios/src/FluentUITester.xcworkspace"
```
Comment (HeyImChris) [inline apps/ios/react-native.config.js:21]: Out of curiosity, what happened to cause it to need to be a relative path now?

File: apps/ios/src/Podfile [hunk 2]
```diff
@@ -1,80 +1,41 @@
...
-  end
-end
+workspace 'FluentUITester.xcworkspace'
 
-target 'FluentUITester' do
-  # YOUR REACT NATIVE PATH (from the iOS folder)
-  rn_path = '../../../node_modules/react-native'
+bundle_script = %{
+export NODE_BINARY=node
+#{__dir__}/../../../node_modules/react-native/scripts/react-native-xcode.sh
```
Comment (HeyImChris) [inline apps/ios/src/Podfile:7]: Save this off into an rn_path var since we use it later below too?

File: apps/ios/src/index.tsx [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
+import { AppRegistry, SafeAreaView } from 'react-native';
 import { ThemeProvider } from '@uifabricshared/theming-react-native';
 import { FabricTester, IFabricTesterProps, customRegistry } from '@fluentui-react-native/tester';
 
 const FluentTester: React.FunctionComponent<IFabricTesterProps> = props => {
   return (
-    <ThemeProvider registry={customRegistry}>
-      <FabricTester {...props} />
-    </ThemeProvider>
+    <SafeAreaView>
```
Comment (HeyImChris) [inline apps/ios/src/index.tsx:10]: Thanks for adding this!

File: apps/ios/react-native.config.js (hunk 1 above)
Comment (tido64) [reply apps/ios/react-native.config.js:21]: I don't think it's ever been relative? In `react-native-test-app`, we've always used it like this. I actually got an error here, which is why I had to change it in the first place.

File: apps/ios/src/index.tsx (hunk 3 above)
Comment (tido64) [reply apps/ios/src/index.tsx:10]: I'll extract this into a separate PR since it's useful regardless.

Comment (tido64) [reply apps/ios/src/index.tsx:10]: Here: #303. I ran `yarn change` but it didn't do anything. Is that normal?

Comment (HeyImChris) [reply apps/ios/src/index.tsx:10]: Yea that's fine. We don't verify change files like how you would if you're using rush in, say, sdx-platform. I don't run yarn change for any changes in this repository

Comment (tido64) [reply apps/ios/src/index.tsx:10]: Your [README](https://github.com/microsoft/fluentui-react-native#beachball) suggests otherwise :stuck_out_tongue:

Comment (HeyImChris) [reply apps/ios/src/index.tsx:10]: Ah yes sorry for the confusion- that applies to Windows. The Apple documentation is lacking right now

**Comment by HeyImChris** [comment]
Windows PR failure is a known issue right now and this PR only affects Apple files and the yarn.lock file. The Apple validation verifies yarn.lock is valid, so pushing this in now to get more progress on the iOS side.

**Comment by Saadnajmi** [comment]
!!! =D 
Excited to see this go in!

**Pull request merged**